travel_platform/skyscanner_data/skyscanner_data.py

```diff
@@ -15,57 +15,41 @@
 inbound_date = config.get('api_skyscanner', 'inbound_date')
 flights_cache_service = FlightsCache(api_key)
 
+#FIXME: why my *args not work?
```

Comment:
Which error type?

FIXME should describe the reason of this FIXME and that's it.

All the errors should be patched in `try .. except` blocks.